packages/react-core/src/components/Toolbar/examples/ToolbarWithFilters.tsx

```diff
@@ -76,30 +82,83 @@ export const ToolbarWithFilters: React.FunctionComponent = () => {
     }
   };
 
-  const onStatusToggle = (_event: any, isExpanded: boolean) => {
-    setStatusIsExpanded(isExpanded);
+  const onStatusToggle = () => {
+    setStatusIsExpanded(!statusIsExpanded);
   };
 
-  const onRiskToggle = (_event: any, isExpanded: boolean) => {
-    setRiskIsExpanded(isExpanded);
+  const onRiskToggle = () => {
+    setRiskIsExpanded(!statusIsExpanded);
   };
 
   const onKebabToggle = (_event: any, isOpen: boolean) => {
     setKebabIsOpen(isOpen);
   };
 
-  const statusMenuItems = [
-    <SelectOption key="statusNew" value="New" />,
-    <SelectOption key="statusPending" value="Pending" />,
-    <SelectOption key="statusRunning" value="Running" />,
-    <SelectOption key="statusCancelled" value="Cancelled" />
-  ];
+  const statusMenuItems = (
+    <SelectList>
+      <SelectOption
+        hasCheckbox
+        key="statusNew"
+        itemId="New"
+        isSelected={filters.status.includes("New")}
+      >
+        New
+      </SelectOption>,
+      <SelectOption
+        hasCheckbox
+        key="statusPending"
+        itemId="Pending"
+        isSelected={filters.status.includes("Pending")}
+      >
+        Pending
+      </SelectOption>,
```

Comment:
If we can remove these commas as part of the build failure fix, otherwise I wouldn't block over getting this in if we can today.